### ISSUE TITLE
big issue

### ISSUE THREAD

**Issue by holydoink** [AHW214/switcher.io#1]
The show Sesame Street – known to millions as being a shining example of televised children's education – has been, since its inception, a vehicle for brainwashing malleable minds and turning them toward a specific purpose. It's not something as mundane or straightforward as social indoctrination, though... and in order to see what's truly going on, we have to go all the way back to 1,500 BC.

The first-ever sundial, created by the ancient Egyptians, was calibrated to include twelve different intervals between sunrise and sunset. Various theories exist for why they chose this number, with the most widely accepted one stipulating that the twelve lunar cycles were their influence. Regardless of this original inspiration, the number twelve became sacred in a way, prompting a secret following of people who took to guiding and shaping the course of humanity's development.

This effect can be traced through the ages with relative ease: The ancient Zoroastrians believed in twelve deity-like figures, and the ancient Babylonians claimed that the heavens were split into twelve sections. The same number was later appropriated by the Jews, who stated that Jacob created twelve tribes of Israel... and that was later taken by the Christians, who said that Yeshua of Nazareth had twelve apostles. The Buddhists revere the number, convinced as they are that a human life comprises twelve stages. Ancient Roman law was based on "The Twelve Tables," and there are twelve signs in the Zodiac. Even Pythagoras used to teach that the number twelve was divine.

It goes even further than that, though:

    There are twelve inches in a foot.
    There are twelve months in a year.
    There are twelve jurors appointed to serve on felony trials.
    The Beatles released twelve studio albums.
    "Force twelve" is the maximum speed of a hurricane.
    Twelve men have walked on the moon.

The number shows up far too frequently to have each manifestation be a coincidence, almost as though some hidden organization is intentionally manufacturing situations in which to prompt its appearance... and one unavowed member of this clandestine group was none other than Jim Henson, the creator of both the Muppets and Sesame Street. He took careful steps to ensure that impressionable youngsters were subtly encouraged to think in Base Twelve, prompting the creation of such songs as "The Ladybugs' Picnic
," or – perhaps too obviously – "Twelve

."

All of this raises one question, though: What is the purpose of this conspiracy? What is being gained from this careful subterfuge? Why did Jim Henson, of all people, take it upon himself to ensure that the number twelve would be indelibly locked in the minds of so many children?

For the answer, you need only count your own ribs, or your own cranial nerves.

Humanity was engineered by beings who came from twelve lightyears away, in the Tau Ceti system. These creatures – our progenitors – left us a clue in our physical forms, and quietly fostered a secret society that was obsessed with injecting the number into every aspect of our lives... and were you to see one of these extraterrestrial entities, you would recognize them immediately

.

TL;DR: The Great Gonzo created humanity, then gave us a way of finding where he came from.
